CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to osbuild-composer
+
+First of all, thank you for taking the time to contribute to osbuild-composer. In this document you will find information that can help you with your contribution.
+
+### Running from sources
+
+We recommend using the latest stable Fedora for development but latest minor release of RHEL/CentOS 8 should work just fine as well. To run osbuild-composer from sources, follow these steps:
+
+```
+$ git clone https://github.com/osbuild/osbuild-composer.git
+$ cd osbuild-composer
+$ dnf group install 'RPM Development Tools' # Install rpmbuild
+$ dnf builddep golang-github-osbuild-composer.spec # Install build-time dependencies
+$ make rpm
+$ sudo dnf install rpmbuild/RPMS/x86_64/* -y
+$ sudo dnf install cockpit-composer -y
+$ sudo systemctl enable --now osbuild-composer.socket
+$ sudo systemctl enable --now cockpit.socket
+```
+
+You can now open https://localhost:9090 in your browser and check the "Image Builder" section.
+
+When developing the code, use `go` executable to build, run, and test you code [1]:
+
+```
+$ go test ./...
+$ go build ./...
+$ go run ./cmd/osbuild-pipeline/
+```
+
+### Testing
+
+See [this document](test/README.md) for more information about testing.
```

Comment:
-`this document` +`README.md`?

Reply:
I think +`test/README.md` makes more sense here, beacuse we also have `README.md` in the root directory and that is not the one I refer to.